src/backend/TrafficCourts/Citizen.Service/Features/Disputes/Create.cs

```diff
@@ -176,7 +180,15 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
 
                     // success, return true
                     activity?.SetStatus(ActivityStatusCode.Ok);
-                    return new Response(submitNoticeOfDispute.EmailVerificationToken);
+
+                    // convert email verification to hashed array of ascii rep of characters
+                    int tokenLength = submitNoticeOfDispute.EmailVerificationToken is not null ? submitNoticeOfDispute.EmailVerificationToken.Length : 0;
+                    int[] intArray = new int[tokenLength];
+                    for (int i=0; i < tokenLength; i++) {
+                        intArray[i] = (int)submitNoticeOfDispute.EmailVerificationToken.ElementAt(i);

```

Comment:
I'm getting null reference warning on this line.

Possible null reference argument for parameter 'source' in 'char Enumerable.ElementAt<char>(IEnumerable<char> source, int index)'

Reply:
if EmailVerificationToken is null on line 185 then tokenLength will be zero and for loop on 187 will not have inside statement executed

Reply:
Yeah I saw that, but the warning still shows up in Studio ... Phil has been trying to fix all compiler warnings so we have a clean IDE. Not a C# expert, but is there a way to fix or suppress the compiler warning?